test/functionality/abstract-interpretation/data-frame/inference.test.ts

```diff
@@ -959,7 +1128,7 @@ df <- aggregate(df, list(group = df$id), mean)
 		]
 	);
 
-	describe.skipIf(skipDplyr)('dplyr Functions', () => {
+	describe.skipIf(skipLibraries)('dplyr Functions', () => {
```

Comment:
is this for debugging? Or for your thesis?  in any way i would document the flag